etcdserver/etcdserverpb/rpc.proto

```diff
@@ -637,6 +638,8 @@ message AuthUserAddResponse {
 
 message AuthUserGetResponse {
   ResponseHeader header = 1;
+
+  repeated string roles = 2;
```

Comment:
I think that if we want the detail of the roles, GetRole() should be called and it would be ok because  GetUser() and GetRole() aren't performance critical. Or are you considering atomicity of the information? Then packing everything in the single RPC call would be suitable.